kimchi/src/tests/lookup.rs

```diff
@@ -465,3 +465,139 @@ fn test_runtime_table_with_more_than_one_runtime_table_data_given_by_prover() {
         "the runtime tables provided did not match the index's configuration"
     );
 }
+
+#[test]
+fn test_runtime_table_only_one_table_with_id_zero_with_non_zero_entries_fixed_values() {
+    let mut rng = rand::thread_rng();
+
+    let first_column = [0, 1, 2, 3, 4, 5];
+    let len = first_column.len();
+
+    let table_id = 0;
+
+    let cfg = RuntimeTableCfg {
+        id: table_id,
+        first_column: first_column.into_iter().map(Into::into).collect(),
+    };
+
+    let data: Vec<Fp> = [0u32, 1, 2, 3, 4, 5].into_iter().map(Into::into).collect();
+    let runtime_table = RuntimeTable {
+        id: table_id,
+        data: data.clone(),
+    };
+
+    // circuit
+    let n_row = 20;
+    let mut gates = vec![];
+    for row in 0..n_row {
+        gates.push(CircuitGate::new(
+            GateType::Lookup,
+            Wire::for_row(row),
+            vec![],
+        ));
+    }
+
+    // witness
+    let witness = {
+        let mut cols: [_; COLUMNS] = array::from_fn(|_col| vec![Fp::zero(); gates.len()]);
+
+        // only the first 7 registers are used in the lookup gate
+        let (lookup_cols, _rest) = cols.split_at_mut(7);
+
+        for row in 0..n_row {
+            // Lookup in table ID 0
+            lookup_cols[0][row] = Fp::zero();
```

Comment:
Maybe ` = Fp::from(table_id)` would be clearer/neater.

Reply:
Will be removed in the refacto suggested in https://github.com/o1-labs/proof-systems/pull/1240/files#r1322922879